css/style.css

```diff
@@ -0,0 +1,15 @@
+h2 {
+  font-family: Arial;
+  padding: 0 20px;
+  background: 	#add8e6;
```

Comment:
Make sure there is a single space between the : and the value

Reply:
Excess "space" removed